src/frontend/components/UI/ThemeSelector/index.tsx

```diff
@@ -68,37 +59,6 @@ export const ThemeSelector = () => {
     getPath()
   }, [])
 
-  // on input change only update the state
-  const onThemesPathChanged = (e: ChangeEvent<HTMLInputElement>) => {
-    if (themesPath !== e.target.value) {
-      setThemesPath(e.target.value)
-    }
-  }
-
-  // only save config on input blur when editing manually
```

Comment:
we lost this feature in the refactor (without this change the config gets saved for every key stroke instead of when blurring the field